sound/soc/sof/intel/hda-dsp.c

```diff
@@ -354,7 +354,7 @@ int hda_dsp_set_power_state(struct snd_sof_dev *sdev,
 	}
 
 	/* Update D0I3C register */
-	value = d0_substate == SOF_DSP_D0I3 ? SOF_HDA_VS_D0I3C_I3 : 0;
+	value = state == SOF_DSP_D0I3 ? SOF_HDA_VS_D0I3C_I3 : 0;
 	snd_hdac_chip_updateb(bus, VS_D0I3C, SOF_HDA_VS_D0I3C_I3, value);
```

Comment:
For D3 entry, Clearing D0I3C.I3 is fine.

Reply:
same comment as above

Reply:
I'll modify this function to do nothing for D3